payments/direct-debit/other-features.md

```diff
@@ -32,6 +32,427 @@ sidebar:
+            "href": "https://api.externalintegration.payex.com/psp/directdebit/payments/3648fa94-7fd8-4e32-a14b-08d608f884ff",
+            "rel": "update-payment-abort"
+        },
+        {
+            "method": "GET",
+            "href": "https://ecom.externalintegration.payex.com/directdebit/payments/sales/993b479653da83671c074316c7455da05fced9d634431edbb64f3c5f80a863f0",
+            "rel": "redirect-sale"
+        }
+    ]
+}
+```
+
+{:.table .table-striped}
+| Property | Description                                                         |
+| :------- | :------------------------------------------------------------------ |
+| `href`   | The target URI to perform the operation against.                    |
+| `rel`    | The name of the relation the operation has to the current resource. |
+| `method` | The HTTP method to use when performing the operation.               |
+
+The operations should be performed as described in each response and not as
+described here in the documentation.
+Always use the `href` and `method` as specified in the response by finding
+the appropriate operation based on its `rel` value.
+The only thing that should be hard coded in the client is the value of the
+`rel` and the request that will be sent in the HTTP body of the request for
+the given operation.
+
+{:.table .table-striped}
+| Operation              | Description                                                                                                                        |
+| :--------------------- | :--------------------------------------------------------------------------------------------------------------------------------- |
+| `update-payment-abort` | Aborts the payment before any financial transactions are performed.                                                                |
+| `redirect-sale`        | Contains the redirect-URI that redirects the consumer to a Swedbank Pay hosted payment page prior to creating a sales transaction. |
+
+### Direct Debit transactions
+
+All Direct Debit specific transactions are described below.
+
+#### Sales
+
+The `Sales` resource lists the sales transactions (one or more) on
+a specific payment.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales HTTP/1.1
+Host: api.externalintegration.payex.com
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
+    "sales": {
+        "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales",
+        "salesList": [
+            {
+                "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales/12345678-1234-1234-1234-123456789012",
+                "selectedBank": "NordeaFI",
+                "deviceIsMobile": "TRUE|FALSE",
```

Comment:
No pipes in code examples, please. Possible values should be described in the property table below the code example.

```suggestion
                "deviceIsMobile": false,
```